### PR TITLE
Ejercicio 1 stat words

### LINKED ISSUE #1: statsWords.sh
Indicador estadístico de longitud de palabras (la más corta, la más larga y el promedio de longitud).

### PR DESCRIPTION
**Pull request by salvanya**

### CODE REVIEW

File: statsWords.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+### to make the average later.
+### Finally we get the word count with the wc -w command and calculate the average. Then the results
+### are printed.
+
+
+FILE=$1
+
+FIRST_ITERATION=true
+
+while read -ra LINE;
```
Comment (fedeantuna) [inline statsWords.sh]:
```suggestion
while read -ra LINE
```

The semicolon is not required

File: statsWords.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+### are printed.
+
+
+FILE=$1
+
+FIRST_ITERATION=true
+
+while read -ra LINE; 
+do
+    for WORD in "${LINE[@]}";
```
Comment (Flor243) [inline statsWords.sh]:
```suggestion
    for WORD in "${LINE[@]}"
```

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+text.txt
```
Comment (Flor243) [inline .gitignore]:
```suggestion
text.txt
test.sh
```

File: statsWords.sh [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            LEAST_CHARACTERS=${#WORD}
+        fi
+        
+        if [ ${#WORD} -gt $MOST_CHARACTERS ]
+        then
+            MOST_CHARACTERS=${#WORD}
+        fi
+
+        CHARACTER_LENGTH_ACCUMULATOR=$(expr $CHARACTER_LENGTH_ACCUMULATOR + ${#WORD})
+    done;
```
Comment (Flor243) [inline statsWords.sh]:
```suggestion
    done
```

**Review by fedeantuna — APPROVED** [review]

**Review by Flor243 — APPROVED** [review]

**Pull request merged**